chromium_src/BUILD.gn

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2018 GitHub, Inc.
+# Use of this source code is governed by the MIT license that can be
+# found in the LICENSE file.
+
+# Builds some of the chrome sources that Electron depends
+# on unconditionally.
+source_set("chrome") {
+  visibility = [ "//electron:electron_lib" ]
+  sources = [
+    "//chrome/browser/ssl/security_state_tab_helper.cc",
+    "//chrome/browser/ssl/security_state_tab_helper.h",
+  ]
+  public_deps = [ "//skia" ]
```

Comment:
why is this here?

Reply:
Without it it errors with,

```
In file included from ../../chrome/browser/ssl/security_state_tab_helper.cc:5:
In file included from ../../chrome/browser/ssl/security_state_tab_helper.h:12:
In file included from ../../content/public/browser/web_contents_observer.h:24:
In file included from ../../third_party/skia/include/core/SkColor.h:11:
In file included from ../../third_party/skia/include/core/SkScalar.h:11:
In file included from ../../third_party/skia/include/core/../private/SkFloatingPoint.h:11:
../../third_party/skia/include/core/../private/../private/SkFloatBits.h:12:10: fatal error: 'SkTypes.h' file not found
#include "SkTypes.h"
         ^~~~~~~~~~~
1 error generated.
```

running `gn path out/Debug //electron/chromium_src:chrome //skia`, resulted in 

```
//electron/chromium_src:chrome --[private]-->
//components/security_state/content:content --[private]-->
//third_party/WebKit/public:blink_headers --[public]-->
//skia:skia
```

Identified the fix through a similar bug in upstream https://bugs.chromium.org/p/chromium/issues/detail?id=702033

Reply:
hah, weird :)

Reply:
... perhaps this should be `deps = [ "//content/public/browser" ]` though?